graphql/schema/rules.go

```diff
@@ -1951,10 +1951,19 @@ func customDirectiveValidation(sch *ast.Schema,
 			for _, h := range forwardHeaders.Children {
 				key := strings.Split(h.Value.Raw, ":")
 				if len(key) > 2 {
-					return append(errs, gqlerror.ErrorPosf(errPos,
+					return append(errs, gqlerror.ErrorPosf(
+						errPos,
+<<<<<<< HEAD
```

Comment:
Seems to be some unresolved conflicts here

Reply:
I messed up big time on this one trying to rebase something. I am fixing this up - give me a few mins

Reply:
sorry for the re-ask .. could you check this one https://github.com/dgraph-io/dgraph/pull/8191 (I have closed #8189 & opened #8190)